src/components/LedgerSignTokenInfo.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState, useEffect} from 'react';
+import { t } from 'ttag';
+
+/**
+ * Component that shows a modal to add one specific unknown token to the wallet
+ *
+ * @memberof Components
+ */
+function LedgerSignTokenInfo({token, showSteps}) {
+
+  const renderTokenUid = () => {
+    // XXX: use if splitting uid on multiple screens
+    // const parts = [];
+    // const numParts = token.uid.length / 8;
+    // token.uid.match(/.{1,8}/g).forEach((part, index) => {
+    //   const i = `(${index+1}/${numParts})`;
+    //   parts.push(<p className="mb-2"><strong>{t`UID`}{i}: </strong>{part}</p>)
+    // });
+
+    // return <div>{parts}</div>
```

Comment:
What's this code?

Reply:
This was a test to split the uid on multiple lines, removed